sound/soc/sof/topology.c

```diff
@@ -1145,19 +1145,22 @@ static int sof_control_load_bytes(struct snd_soc_component *scomp,
 	struct snd_soc_tplg_bytes_control *control =
 		container_of(hdr, struct snd_soc_tplg_bytes_control, hdr);
 	struct soc_bytes_ext *sbe = (struct soc_bytes_ext *)kc->private_value;
-	int max_size = sbe->max;
+	size_t max_size = sbe->max, priv_size = le32_to_cpu(control->priv.size);
```

Comment:
would put this comma separated definition/assignment to separate line

Reply:
+1, I don't think we use commas anywhere